.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+name-template: $NEXT_PATCH_VERSION
+tag-template: $NEXT_PATCH_VERSION
+version-template: $MAJOR.$MINOR-beta-$PATCH
+
+# Emoji reference: https://gitmoji.carloscuesta.me/
+categories:
+  - title: 🚨 Removed
+    label: removed
+  - title: ⚠️ Deprecated
+    label: deprecated
+  - title: 🚀 New features and improvements
+    label: enhancement
+  - title: 🐛 Bug Fixes
+    label: bugfix
+  - title: 📝 Documentation updates
+    label: documentation
+  - title: 📦 Dependency updates
+    label: dependencies
+  - title: 🚦 Internal changes
+    label: internal
+exclude-labels:
+  - reverted
+  - no-changelog
+  - skip-changelog
+  - invalid
+
+template: |
+  ## Changes
+  $CHANGES
+
+replacers:
+  - search: '/\[*JENKINS-(\d+)\]*\s*-*\s*/g'
```

Comment:
Gotta be careful these might not always match, depending on how users write their PR titles.

Reply:
Right. We just need to start from something. I can also add a straightforward `JENKINS-123` replacement